core/org.eclipse.cdt.core/parser/org/eclipse/cdt/internal/core/dom/parser/cpp/semantics/EvalPackAccess.java

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+* Copyright (c) 2016 Institute for Software, HSR Hochschule fuer Technik
```

Comment:
I was intending to keep original attribution to the authors of `EvalCompositeAccess` + add `and others` clause because this new subclass is largely a copy and cut - but apparently that slipped.
Fixed now, hope I did this attribution change correctly.